controllers/subscription_controller.go

```diff
@@ -290,6 +291,22 @@ func (r *SubscriptionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
+	err := mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
+
+		odfDepsSub := GetStorageClusterSubscriptions()[0]
+
+		if odfDepsSub.Spec.Package != OdfDepsSubscriptionPackage {
+			panic("odfDepsSub variable is expected to contain the 'odf-dependencies' subscription. " +
+				"Ensure the 'odf-dependencies' subscription indexed at 0.")
+		}
+
+		return EnsureDesiredSubscription(r.Client, odfDepsSub)
+
```

Comment:
pls remove extra lines, all actions seems to be a single unit.

Reply:
Done